### DISCUSSION TITLE
Device discovered as "Other: 33" | Zone motors

### DISCUSSION THREAD

**Discussion by GammaScorpii** [omerfaruk-aran/esphome_samsung_hvac_bus#65] [General]
```
[17:16:19][C][samsung_ac:041]: Configured devices:
[17:16:19][C][samsung_ac:061]: Discovered devices:
[17:16:19][C][samsung_ac:062]:   Outdoor: c8
[17:16:19][C][samsung_ac:063]:   Indoor:  00
[17:16:19][C][samsung_ac:065]:   Other:   33
```

A two part question, what could device 33 be?

And could it be my Zone motors? Whoever installed my ducted non-NASA (I think) system, also installed a separate unbranded control panel to control the fan speed in individual rooms or zones. Each room can be set to OFF LOW MED HIGH. And there's a boost function as well. The only thing is it's possible the zone motors and the samsung unit are not connected?

Here's some pictures:

Is there a way to interface with this? Even if it requires separate hardware. I do note that the switch looking thing as an additional port on the same side as the control panel is plugged into, could I add something here?

![PXL_20240127_113014855 (Custom)](https://github.com/lanwin/esphome_samsung_ac/assets/28531783/a6d6d88a-4759-4a17-b3b6-8e5c728d7300)
![PXL_20240127_113030753 (Custom)](https://github.com/lanwin/esphome_samsung_ac/assets/28531783/ae302799-8288-463d-84f1-18bbc352bbcc)
![PXL_20240127_113037049 (Custom)](https://github.com/lanwin/esphome_samsung_ac/assets/28531783/21296b0e-6463-460c-b250-29a4a768b029)
The motor itself fitted on the tubing:
![PXL_20240127_113215870 (Custom)](https://github.com/lanwin/esphome_samsung_ac/assets/28531783/dd2c7d2f-75e8-4abf-87bb-815211bb7a72)
It's almost impossible to find any info on it. I'm pretty sure it's this company that makes (sells) it
https://www.airdiffusion.com.au/

**Comment by lanwin** [comment]
I would say yes. Everyone sitting in the bus has an address. So I would expect that you are right and it is indeed the zone controller.

**Reply by GammaScorpii** [reply]
But it's not possible to control it right? I mean maybe possible, but not until some work has gone into decoding the messages, etc?

**Reply by lanwin** [reply]
Its complicated. I guess its controllable and I could sent messages to it. The Problem is that I did not know what messages I need to send and what data they should contain. 

Since we have no documentation, the only option is to watch what messages getting sent over the bus. So when one of the other devices or even an AC remote is sending data to this, we could try it. 

You could enable debug_log_messages from here https://github.com/lanwin/esphome_samsung_ac?tab=readme-ov-file#development and then watch if you could see messages that have 33 as src or dst.

**Comment by tech2urdoor** [comment]
I had the exact same discovered devices! I also have a seperate controller for the zones. Although I can't change fan speeds for those zones. The zonemaster controller only allows me to turn on or off 4 different zones.
